admin/services/check-start.service.js

```diff
@@ -227,9 +227,14 @@ checkStartService.prepareCheckQueueMessages = async function (checkIds) {
   const checks = await checkFormAllocationDataService.sqlFindByIdsHydrated(checkIds)
   const sasExpiryDate = moment().add(config.Tokens.sasTimeOutHours, 'hours')
 
+  const hasLiveChecks = R.all(c => R.equals(c.check_isLiveCheck, true))(checks)
```

Comment:
my only concern here would be if there is a situation where there could be a mix of live/practice checks?  although my understanding is that a batch will always be all practice or all live?

Reply:
@GuyHarwood as far as I see this is being called when familiarisation pins are generated and when live pins are generated.
It appears that with the current implementation we will never have a mixture of live/practice checks at that level.